cmd/carbonapi/main.go

```diff
@@ -64,5 +64,6 @@ func main() {
 	if err != nil {
 		logger.Error("Error initializing app")
 	}
-	app.Start()
+	stop := app.Start()
```

Comment:
I think we should rename _stop_ -> _flush_

Reply:
Done